CleverCloud/clever-components
# Web dev server: upgrade to `0.3.0` and switch back to `nodeResolve: true`

Fixes #812 

## What does this PR do?

- Upgrades the WDS dependency to `0.3.0`,
- Changes `nodeResolve` to `true` now that `exportConditions` is set to `development` by default.

## How to review?

2 reviewers should be enough.

- Check both commits,
- Run locally and check that you still have the Lit warning in your console when viewing a story.
![Lit warning: Lit is in dev mode. Not recommended for production! See https://lit.dev/msg/dev-mode for more information.](https://github.com/CleverCloud/clever-components/assets/100240294/da679404-b57c-4461-a48b-3f883b7aef37)
